.gitignore

```diff
@@ -1,8 +1,2 @@
-pkg
-bin
-hound.sublime-workspace
-data
 .vagrant
-pub/index.html
-pub/excluded_files.html
-config.json
```

Comment:
Oh great point, `make clean` really shouldn't delete it. That was only useful while I was getting it working.